### PR TITLE
Use version from VERSION for `app -v` and macOS bundle version

### PR DESCRIPTION
**Pull request by jelmervdl**
Use version from VERSION for `app -v` and macOS bundle version. I've bumped the version in VERSION to match the one that was hard-coded in the application. This should solve #20.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -75,3 +75,4 @@ build*/
 
 # Qt creator:
 CMakeLists.txt.user
+/src/version.h
```
Comment (XapaJIaMnu) [inline .gitignore:78]: This should go into `CMAKE_CURRENT_BINARY_DIR` not the source dir

File: CMakeLists.txt [hunk 2]
```diff
@@ -113,6 +113,9 @@ set(PROJECT_SOURCES
         ${TS_FILES}
 )
 
+# Make PROJECT_VERSION_* available to our code as TRANSLATELOCALLY_VERSION_*
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/version.h.in
```
Comment (XapaJIaMnu) [inline CMakeLists.txt:117]: The generated file should go into the binary dir, not the source dir.

**Review by XapaJIaMnu — CHANGES_REQUESTED** [review]
THank you for fixing this! I think the generated files at compile time should go into the binary dir.

**Comment by jelmervdl** [comment]
I agree. I copied what Marian did, but the cmake documentation also seems to recommend using `${CMAKE_CURRENT_BINARY_DIR}`.

**Pull request merged**